djangogirls/blog/admin.py

```diff
@@ -0,0 +1 @@
+"""Configuration for blog admin."""
```

Comment:
I don't see any code adding the model to the admin panel. Where is it?

Reply:
In the next part of the tutorial.